postgres/metadata.csv

```diff
@@ -78,5 +78,9 @@ postgresql.queries.local_blks_dirtied,count,,block,,"Total number of local block
 postgresql.queries.local_blks_written,count,,block,,"Total number of local blocks written per normalized query, database, and user. (DBM only)",0,postgres,postgres queries local blocks written
 postgresql.queries.temp_blks_read,count,,block,,"Total number of temp blocks read per normalized query, database, and user. (DBM only)",0,postgres,postgres queries temp blocks read
 postgresql.queries.temp_blks_written,count,,block,,"Total number of temp blocks written per normalized query, database, and user. (DBM only)",0,postgres,postgres queries temp blocks written
+postgresql.queries.duration.max,gauge,,,,The age of the longest running query per user, db, app and host (DBM only).,0,postgres,postgres queries duration max
```

Comment:
To be consistent with the other query metrics, we should refer to the database as "database" instead of "db". And instead of app "app" probably "application"